frontend/src/components/form/Form.tsx

```diff
@@ -0,0 +1,159 @@
+import {
+  Form as AForm, Input, Row, Typography,
+} from 'antd';
+import styled from 'styled-components';
+
+import { Button } from '@/components/buttons';
+import { Colors } from '@/styles/colors';
+
+const StyledForm = styled(AForm)`
+  background: ${Colors.onPrimary};
+  border: 1px solid ${Colors.shadow};
+  padding: 20px;
+  border-radius: 10px;
+
+  .first-and-last-name-form {
+      display: flex;
+      justify-content: space-between;
+      gap: 30px;
+  }
+
+  .first-and-last-name-form > * {
+    width: 100%;
+  }
+
+  .first-and-last-name-form Input {
+    width: 100%;
+  }
+
+  .form-item.label {
+    border: 1px blue solid;
+  }
+
+  .submit-btn-parent {
+    width: 100%;
+  }
+
+  Input {
+    border-radius: 5px;
+  }
+`;
+
+const Form: React.VFC = () => {
+  const [form] = StyledForm.useForm();
+  const { Title } = Typography;
```

Comment:
Trochę dodałbym wolnych linii, za bardzo jest to zbite w kupę.

const [form] = StyledForm.useForm();
[wolna linia]
const { Title } = Typography;
[wolna linia]
return (...